Add-On UI/scripts/process_query.js

```diff
@@ -0,0 +1,236 @@
+//json object that contains all aspects and table's data
+var JsonObj = {
+  'intent': 'null',
+  'table': 'null',
+  'rowRange': 'null',
+  'metric': 'null',
+  'dimensions': 'null',
+  'summaryOperator': 'null',
+  'isAsc': 'null',
+  'k': 'null',
```

Comment:
'k' also doesnt really tell a lot about the argument.

Reply:
Should I write limit or something other name ?